pages/_app.tsx

```diff
@@ -82,6 +83,25 @@ export default function App({ Component, pageProps, router }: AppProps) {
         <link href="/manifest.json" rel="manifest" />
         <meta content="#1667C2" name="theme-color" />
       </Head>
+
+      <Script
+        dangerouslySetInnerHTML={{
+          __html: `(function(w,d,s,l,i){w[l] = w[l] || [];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-5X4ZPBX');`,
+        }}
+      />
+      <noscript>
+        <iframe
+          height="0"
+          src="https://www.googletagmanager.com/ns.html?id=GTM-5X4ZPBX"
+          style={{ display: "none", visibility: "hidden" }}
+          title="gtm"
+          width="0"
+        />
```

Comment:
need to add </iframe> instead for the noscript to work properly. 

See previous implementation here: https://github.com/kawalcovid19/wargabantuwarga.com/pull/36/files?file-filters%5B%5D=.tsx

Reply:
in React, they should be identical, Mas @baraeb92.

But somehow it's not showing in the page now, could you please investigate it further, Mas @mazipan?

Reply:
Bisa insert expectation kode komplit nya dari GTM gak ya, Mas @baraeb92.?

Reply:
Since all this code are just `copy-paste` from previous implementation without any modifications

Reply:
Since all this code are just copy-paste from previous implementation without any modifications

Reply:
![Screen Shot 2021-07-26 at 10 50 41](https://user-images.githubusercontent.com/7221389/126930683-0943086e-34ac-4aac-9477-a1104d9138f0.png)

It already exist, it just under `main` tag, should we move it under `body`?